kong/plugins/file-log/schema.lua

```diff
@@ -1,9 +1,11 @@
-local pl_utils = require "pl.utils"
+local pl_file = require "pl.file"
+local pl_path = require "pl.path"
 
 local function validate_file(value)
-  local ok = pl_utils.executeex("touch "..value)
-  if not ok then
-    return false, "Cannot create file. Make sure the path is valid, and has the right permissions"
+  -- create file in case it doesn't exist
+  if not pl_path.exists(value) then
```

Comment:
If that's the case, we use `pl_path.exists(..)` quite a lot to check for the existence of files pretty much all over Kong. It will have to be replaced everywhere.